packages/eslint-plugin-i18n/package.json

```diff
@@ -29,7 +17,18 @@
     "mversion": "^1.10.1",
     "sinon": "^3.2.1"
   },
-  "dependencies": {
-    "minimatch": "3.0.4"
+  "peerDependencies": {
+    "eslint": "^4.0.0"
```

Comment:
Not sure it's useful for deps/peer dependencies. But maybe we could fix them ? Since `^` allows for the upgrade of the dependencies on breaking changes version value :/